src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -904,12 +903,15 @@ export default class Aircraft {
 
         let isExpeditingString = '';
         if (expedite) {
-            isExpeditingString = 'and expedite';
+            isExpeditingString = ' and expedite';
         }
 
+        const newAltitude = this.fms.altitudeForCurrentWaypoint();
+        const radioTrendAltitude = radio_trend('altitude', this.altitude, newAltitude);
+        const currentWaypointRadioAltitude = radio_altitude(newAltitude);
         const readback = {
-            log: `${radioTrendAltitude} ${this.fms.altitudeForCurrentWaypoint()} ${isExpeditingString}`,
-            say: `${radioTrendAltitude} ${currentWaypointRadioAltitude} ${isExpeditingString}`
+            log: `${radioTrendAltitude} ${newAltitude}${isExpeditingString}`,
+            say: `${radioTrendAltitude} ${currentWaypointRadioAltitude}${isExpeditingString}`
```

Comment:
@erikquinn here is another possible solution, it adds a little more code.  But I think it's cleaner. you could always just build the base string with `${radioTrendAltitude} ${currentWaypointRadioAltitude}` and the other one before the `if (expedite)` block and simply overwrite that string when expedite is true?
```javascript
const radioTrendWithAltitude = `${radioTrendAltitude} ${newAltitude}`;
const readback = {
    log: radioTrendWithAltitude
}

if (expedite) {
    readback.say = `${radioTrendWithAltitude} and expedite`;
}
```
_obviously this is an abbreviated example. I'm sure you can see where I'm going with this_